### PR TITLE
Monkey patch for bug in safari 14

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

Work around for a bug that is shipped with Safari 14 with `delegatesFocus` it is fixed in the beta of Safari 14 but they decided to ship the version with the bug :tada:

https://bugs.webkit.org/show_bug.cgi?id=215622

## Type of change

- [X] Fixing the fuckups from Safari
- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by balloob — APPROVED** [review]

File: src/entrypoints/core.ts [hunk 1]
```diff
@@ -32,6 +32,20 @@ declare global {
   }
 }
 
+// https://github.com/home-assistant/frontend/pull/7031
+const isSafari14 = /^((?!chrome|android).)*version\/14\.0.*safari/i.test(
```
Comment (balloob) [inline src/entrypoints/core.ts:36]: Why the guard for Chrome / Safari ?

File: src/entrypoints/core.ts [hunk 2]
```diff
@@ -1,5 +1,4 @@
 // Compat needs to be first import
-import "../resources/compatibility";
```
Comment (balloob) [inline src/entrypoints/core.ts:2]: This needs to be the first import, as per the comment…

File: src/entrypoints/core.ts (hunk 1 above)
Comment (bramkragten) [reply src/entrypoints/core.ts:36]: Chrome/android also has safari in the user string

Comment (bramkragten) [reply src/entrypoints/core.ts:36]:
```
"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.102 Safari/537.36"
```

File: src/entrypoints/core.ts (hunk 2 above)
Comment (bramkragten) [reply src/entrypoints/core.ts:2]: Stupid auto import organizer... :-/

File: src/entrypoints/core.ts (hunk 1 above)
Comment (balloob) [reply src/entrypoints/core.ts]: lol that string is so useless.

**Review by balloob — APPROVED** [review]

**Comment by balloob** [comment]
LOOKS GOOD GO GO GO 🏁

**Pull request merged**